app/src/androidTest/java/net/aiscope/gdd_app/SampleDtoKtTest.kt

```diff
@@ -0,0 +1,28 @@
+package net.aiscope.gdd_app

```

Comment:
Move it to Sample's package? (package net.aiscope.gdd_app.model)

Reply:
Done